internal/pkg/cmd/flags.go

```diff
@@ -16,6 +16,8 @@ import (
 	"github.com/confluentinc/cli/internal/pkg/utils"
 )
 
+var serializationFormats = []string{"string", "avro", "integer", "jsonschema", "protobuf"}
```

Comment:
can you explain the difference between `serializationFormats` and `schemaBasedFormats`? confused why there's this `integer` format, is that only used for key schema?

Reply:
but i see it's being used in the flag definition of both key and value schema format

Reply:
`schemaBasedFormats` is a subset of `serializationFormats`. Not all serialization formats involve schemas.

> key and value schema format

I think you mean "serialization format" here. Using one of the `--key-format` and `--value-format` flags doesn't necessarily mean there is a schema involved.